examples/visualisation/visualise_data.ipynb

```diff
@@ -311,6 +313,13 @@
     "    clear_output(wait=True)\n",
     "    display(PIL.Image.fromarray(im[::-1]))"
    ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
sorry I just noticed you may have another empty cell here :(

Reply:
don't you dare apologise for my mistakes :D  it's fixed with the lastest commit